pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+# include = ["scout"]
+namespaces = false
+
+[project]
+name = "scout"
+readme = "README.md"
+version = "0.9"
+authors = [
+  {email = "chioke.harris@nrel.gov"},
+  {name = "Chioke Harris"},
+  {email = "jared.langevin@lbl.gov"},
+  {name = "Jared Langevin"}
+]
+requires-python = ">=3.8"
+license = {text = "Apache-2.0"}
```

Comment:
Change the license reference to [LICENSE.md](https://github.com/trynthink/scout/blob/master/LICENSE.md).